<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BWM Voice Operator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPlainTextEdit" name="enterText">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(186, 189, 182);
font: 300 14pt &quot;URW Bookman&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="imageLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="volume">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Volume: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="volume_2">
          <property name="styleSheet">
           <string notr="true">gridline-color: rgb(138, 226, 52);</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Rate:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="rate">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Pitch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="pitch">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="voice_2">
          <property name="text">
           <string>Voices</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="voice">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(211, 215, 207);
font: 700 12pt &quot;Ubuntu Condensed&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="speakReady">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(115, 210, 22);</string>
        </property>
        <property name="text">
         <string>Speak</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
